app/Http/Controllers/Legacy/Web/CampaignsController.php

```diff
@@ -16,7 +16,7 @@ class CampaignsController extends Controller
     public function __construct()
     {
         $this->middleware('auth', ['only' => ['store', 'update', 'destroy']]);
-        $this->middleware('role:admin,staff', ['only' => ['store', 'update', 'destroy']]);
+        $this->middleware('role:admin,staff');
```

Comment:
I think we want all staffers & admins to be able to view these but only allow admins to edit… so:

```php
$this->middleware('auth');
$this->middleware('role:admin,staff', ['only' => ['index', 'show']]);
$this->middleware('role:admin', ['only' => ['create', 'store', 'edit', 'update', 'destroy']]);

```